### PR TITLE
Ref Guide - admin-menu

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/10587

### CODE REVIEW

File: modules/reference/pages/admin/admin-menu.adoc [hunk 1]
```diff
@@ -1,8 +1,10 @@
 = Admin Menu
 
-The menu:Main Menu[Admin] pages allows {productname} customers to manage the basic configuration, including creating and managing multiple organizations.
+The [guimenu]``Admin`` menu provides access to features for managing {productname} configuration.
```
Comment (keichwa) [inline modules/reference/pages/admin/admin-menu.adoc]:
```suggestion
The menu:Admin[] menu provides access to features for managing {productname} configuration.
```

File: modules/reference/pages/admin/admin-menu.adoc [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
+Configuration tasks include creating and managing organizations, users, and tasks.
+You can also use the setup wizard to help configure {productname}.
 
-[TIP]
+[NOTE]
 ====
-Only the {productname} administrator can access the menu:Main Menu[Admin] pages.
+The [guimenu]``Admin`` menu is only available if you are logged in with a {productname} administrator account.
```
Comment (keichwa) [inline modules/reference/pages/admin/admin-menu.adoc]:
```suggestion
The menu:Admin[] menu is only available if you are logged in with a {productname} administrator account.
```

File: modules/reference/pages/admin/admin-menu.adoc (hunk 1 above)
Comment (Loquacity) [reply modules/reference/pages/admin/admin-menu.adoc]: I thought we decided against doing this?

Comment (keichwa) [reply modules/reference/pages/admin/admin-menu.adoc]: Maybe, you are right. Only if there are more than one entries using menu: ? I think we did not talk abut this (maybe, I forgot it).

Comment (Loquacity) [reply modules/reference/pages/admin/admin-menu.adoc]: I might have imagined it :wink:

**Pull request merged**